cmake/UnitTests.cmake

```diff
@@ -2377,6 +2397,11 @@ string(REGEX REPLACE ";" "\$|" IGPU_LEVEL0_FAILED_TESTS_STR "${IGPU_LEVEL0_FAILE
 string(REGEX REPLACE ";" "\$|"    CPU_POCL_FAILED_TESTS_STR "${CPU_POCL_FAILED_TESTS}")
 string(REGEX REPLACE ";" "\$|" ALL_FAILED_TESTS_STR "${ALL_FAILED_TESTS}")
 
+if(NOT DEFINED PARALLEL_TESTS)
+  set(PARALLEL_TESTS 1)
+endif()
+
+set(TEST_OPTIONS -j ${PARALLEL_TESTS} --timeout 120 --output-on-failure)
 add_custom_target(check COMMAND ${CMAKE_CTEST_COMMAND} ${TEST_OPTIONS} -E ${ALL_FAILED_TESTS_STR} VERBATIM)
```

Comment:
make check is pretty much deprecated in favor for `scripts/check.py` because setting this option would force a reconfigure.

Reply:
ok, removed the commit